mlrun/projects/pipelines.py

```diff
@@ -984,7 +984,7 @@ def github_webhook(request):
     return {"msg": "pushed"}
 
 
-def load_and_run(
```

Comment:
add alias function to load_and_run that points to load_and_run_workflow with relevant flags